src/testing/jest/jest-27-and-under/jest-runner.ts

```diff
@@ -60,7 +60,7 @@ export function createTestRunner(): any {
       // filter out only the tests the flags said we should run
       tests = tests.filter((t) => includeTestFile(t.path, env));
 
-      if (env.__STENCIL_SCREENSHOT__ === 'true') {
```

Comment:
Add this check since `JSON.parse` would [fail here](https://github.com/ionic-team/stencil/pull/4890/files#diff-7cec921f2833685474ae4b5c745bb512842c238e18cd9c729f9db6de28d81f5bL69) if `__STENCIL_EMULATE_CONFIGS__` was `undefined | null` or a falsy string.  This check doesn't handle the case of malformed JSON, as I was only looking at SNC violation fixes here